ghost/i18n/locales/pt/newsletter.json

```diff
@@ -1,28 +1,28 @@
 {
     "{date}": "{date}",
-    "Become a paid member of {site} to get access to all premium content.": "",
-    "By {authors}": "",
-    "Comment": "",
-    "complimentary": "",
-    "Email": "",
-    "free": "",
-    "Keep reading": "",
-    "Less like this": "",
-    "Manage subscription": "",
-    "Member since": "",
-    "More like this": "",
-    "Name": "",
-    "paid": "",
-    "Subscription details": "",
-    "trialing": "",
-    "Unsubscribe": "",
-    "Upgrade": "",
-    "Upgrade to continue reading.": "",
-    "View in browser": "",
-    "You are receiving this because you are a <strong>%%{status}%% subscriber</strong> to {site}.": "",
-    "Your free trial ends on {date}, at which time you will be charged the regular price. You can always cancel before then.": "",
-    "Your subscription has been canceled and will expire on {date}. You can resume your subscription via your account settings.": "",
-    "Your subscription has expired.": "",
-    "Your subscription will expire on {date}.": "",
-    "Your subscription will renew on {date}.": ""
+    "Become a paid member of {site} to get access to all premium content.": "Torne-se um membro pago de {site} para aceder a todo o conteúdo exclusivo.",
+    "By {authors}": "Por {authors}",
+    "Comment": "Comentar",
+    "complimentary": "oferta",
+    "Email": "E-mail",
+    "free": "grátis",
+    "Keep reading": "Continuar a ler",
+    "Less like this": "Não gostei",
+    "Manage subscription": "Gerir subscrição",
+    "Member since": "Membro desde",
+    "More like this": "Gostei",
+    "Name": "Nome",
+    "paid": "pago",
+    "Subscription details": "Detalhes da subscrição",
+    "trialing": "em període de avaliação",
+    "Unsubscribe": "Cancelar subscrição",
+    "Upgrade": "Fazer upgrade",
+    "Upgrade to continue reading.": "Faça upgrade para continuar a ler.",
+    "View in browser": "Ver no navegador",
+    "You are receiving this because you are a <strong>%%{status}%% subscriber</strong> to {site}.": "Está a receber este e-mail porque é <strong>subscritor/a %%{status}%%</strong> de {site}.",
```

Comment:
The 'status' variable will be substituted with the strings for free, paid, complimentary, and trialing. Could you please review these for grammar? This is an area that has been troublesome for many translators.  AI suggests 'com oferta' instead of 'oferta'.

Reply:
I changed the phrasing. It's more verbose, but I think clearer and a but more natural.